.travis.yml

```diff
@@ -23,7 +23,8 @@ before_script:
   - python setup.py sdist install
 
 script:
-  - docker-compose -f docker/docker-compose.yml up --build --abort-on-container-exit
-  - exitcode=$(docker inspect -f '{{.State.ExitCode}}' docker_maprtest_1)
-  - docker-compose -f docker/docker-compose.yml rm -fv
-  - exit $exitcode
\ No newline at end of file
+  - echo 'Tested in CI'
```

Comment:
If it's tested in CI, and no longer using travis, then why not delete .travis.yml entirely and disable travis for this repo?